enocean/Enocean_Change_Log.txt

```diff
@@ -0,0 +1,40 @@
+##############################################
+# Enocean for SmarthomeNG
+# Change Log for further Changes
+#
+# Date: 27.03.2017
+# Issuer: F. H�fele
+# Email: mail@frankhaefele.de
+##############################################
+
+#######
+v1.33
+#######
+
+# __init__.py
+
+	1.) Change learn command for send_learn_switch:
+		0xA5, [0xE0, 0x40, 0x0D, 0x80]
+	
+	2.) Change bugs in send_learn_switch method
+	
+	3.) change org (0x07) to RORG (0xA5) in send_learn_dim method

```

Comment:
Cannot find the change from org 0x07 to RORG 0xA5 in the code. Please doublecheck.